text-kit/autogen.kit.d/base.yml

```diff
@@ -25,6 +29,81 @@ dev_github:
               cmake_src_configure
             }
 
+    - qpdf:
+        github:
+          user: qpdf
+        transform:
+          - kind: string
+            match: 'v '
+            replace: ''
+          - kind: string
+            match: 'release-qpdf-'
+            replace: ''
+        assets:
+          - name: '{{ .Values.pn }}-{{ .Values.version }}.tar.gz'
+            matcher: '{{ .Values.pn }}-{{ .Values.version }}.tar.gz'
+            url: 'https://github.com/qpdf/qpdf/releases/download/v{{ .Values.version }}/{{ .Values.pn }}-{{ .Values.version }}.tar.gz'
+          - name: '{{ .Values.pn }}-{{ .Values.version }}-doc.zip'
+            url: 'https://github.com/qpdf/qpdf/releases/download/v{{ .Values.version }}/{{ .Values.pn }}-{{ .Values.version }}-doc.zip'
+            final_name: '{{ .Values.pn }}-{{ .Values.version }}-doc.zip'
```

Comment:
This attribute doesn't exist. The name is directly take from the `name` attribute